driverapi/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(
+    DRIVERAPI_SOURCES
+    src/librecuda.cpp
+    src/librecuda_status.cpp
+    src/cmdqueue.cpp
+)
+
+add_library(driverapi SHARED ${DRIVERAPI_SOURCES})
+target_include_directories(
+    driverapi PUBLIC 
+    "include/"
+    "internal/")
```

Comment:
internal/ still PUBLIC? xD

Reply:
ohhh, sry, fix it right now🥺🥺

Reply:
Now seems OK.

Reply:
https://github.com/mikex86/LibreCuda/pull/11/files#diff-7beb39f55cd695b195e9fa836b6164aa41b6be973882caf9c0d7a417082cf006R8-R11